spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -397,6 +397,9 @@
       "person_or_org": {
         "name": organization_name,
         "type": "organizational"
+      },
+      "role": {
+        "id": "role-other"
```

Comment:
The memoized data from the latest export included this so I added it to the spec.